src/components/Carousel/Carousel.tsx

```diff
@@ -127,17 +132,17 @@ const Carousel: (props: { children?: ReactElement[] }) => JSX.Element = ({
     <ButtonContainer>
       {leftVisible && (
         <Previous onClick={previous}>
-          <ArrowLeftCircle size={`calc(${theme.typography.fontSize} * 2)`} />
+          <ArrowLeftCircle size={`calc(${theme.typography.fontSize} * 2)`} role='img' aria-label={accessibility?.previousCarouselButtonLabel || 'Previous slides'} focusable='false'/>
         </Previous>
       )}
-      <ItemContainer ref={ref.container}>
+      <ItemContainer ref={ref.container} role='group' aria-roledescription={accessibility?.carouselRoleDescription || 'Carousel'}>
         {children?.map((child, i) =>
-          React.cloneElement(child, { ref: ref.items[i] }, undefined),
+          React.cloneElement(child, { ref: ref.items[i], roleDescription: accessibility?.slideRoleDescription }, undefined),
         )}
       </ItemContainer>
       {rightVisible && (
         <Next onClick={next}>
-          <ArrowRightCircle size={`calc(${theme.typography.fontSize} * 2)`} />
+          <ArrowRightCircle size={`calc(${theme.typography.fontSize} * 2)`} role='img' aria-label={accessibility?.previousCarouselButtonLabel || 'Next slides'} focusable='false'/>
```

Comment:
Hi @correi-f it should be `nextCarouselButtonLabel` there - probably a copy-paste mistake ;)

Reply:
Done